### PR TITLE
update tiup doc

### PR DESCRIPTION
**Pull request by kaaaaaaang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -118,6 +120,36 @@ server_configs:
 
 上述配置指定了 TiDB 和 TiKV 的全局配置。
 
+### `component_versions`
+
+> **注意：**
+>
+> 对于 TiDB、TiKV、PD、TiCDC 等本身共用版本号的组件。没有完整的测试来保证它们能够在跨版本混部的场景下能够正常工作。请仅用于测试场景或者在 support 的帮助下进行操作。
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:127]:
```suggestion
> 对于 TiDB、TiKV、PD、TiCDC 等本身共用版本号的组件，没有完整的测试来保证它们能够在跨版本混部的场景下能够正常工作。请仅用于测试场景，或者在技术支持的帮助下进行操作。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -118,6 +120,36 @@ server_configs:
...
+
+`component_versions` 用于指定某个组件的版本号。当不配置 component_versions 时，各个组件要么使用和 TiDB 集群一致的版本号（如 pd、tikv），要么使用最新版（如 alertmanager）。当配置该字段时，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。用户应当只在需要特别固定某个组件的版本号的时候才进行相应配置。主要包含以下字段：
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:129]:
```suggestion
`component_versions` 用于指定某个组件的版本号。如果没有配置 `component_versions`，各个组件要么使用和 TiDB 集群一致的版本号（如 PD、TiKV），要么使用最新版本号（如 Alertmanager）。如果配置了该字段，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。请仅在需要使用某个固定组件的版本号时进行相应配置。主要包含以下字段：
```

File: tiup/tiup-cluster-topology-reference.md [hunk 3]
```diff
@@ -118,6 +120,36 @@ server_configs:
...
+
+- `tikv`：TiKV 组件的版本
+- `tiflash`：TiFlash 组件的版本
+- `pd`：PD 组件的版本
+- `tidb_dashboard`： 独立部署的 TiDB Dashboard 组件的版本
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:134]:
```suggestion
- `tidb_dashboard`：独立部署的 TiDB Dashboard 组件的版本
```

File: upgrade-tidb-using-tiup.md [hunk 4]
```diff
@@ -194,6 +194,27 @@ tiup cluster upgrade <cluster-name> v7.4.0
...
+从 tiup-cluster v1.14.0 开始，支持在升级集群的时候指定其中某些组件到特定版本。指定的这些组件会被固定到指定的版本，在下次升级中不会改变版本号，除非再次指定版本。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster upgrade -h | grep "version string"
+      --alertmanager-version string        Fix the version of alertmanager and no longer follows the cluster version.
+      --blackbox-exporter-version string   Fix the version of blackbox-exporter and no longer follows the cluster version.
+      --cdc-version string                 Fix the version of cdc and no longer follows the cluster version.
+      --ignore-version-check               Ignore checking if target version is bigger than current version
```
Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:208]:
```suggestion
      --ignore-version-check               Ignore checking if target version is bigger than current version.
```

Comment (kaaaaaaang) [reply upgrade-tidb-using-tiup.md:208]: these lines are copied from promgram output,I think we should not change it until we fix it in program

File: upgrade-tidb-using-tiup.md [hunk 5]
```diff
@@ -194,6 +194,27 @@ tiup cluster upgrade <cluster-name> v7.4.0
...
+      --node-exporter-version string       Fix the version of node-exporter and no longer follows the cluster version.
+      --pd-version string                  Fix the version of pv and no longer follows the cluster version.
```
Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:210]:
```suggestion
      --pd-version string                  Fix the version of pd and no longer follows the cluster version.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 6]
```diff
@@ -118,6 +120,36 @@ server_configs:
 
 上述配置指定了 TiDB 和 TiKV 的全局配置。
 
+### `component_versions`
+
+> **注意：**
+>
+> 对于 TiDB、TiKV、PD、TiCDC 等本身共用版本号的组件，没有完整的测试来保证它们能够在跨版本混部的场景下能够正常工作。请仅用于测试场景，或者在技术支持的帮助下进行操作。
+
+`component_versions` 用于指定某个组件的版本号。如果没有配置 `component_versions`，各个组件要么使用和 TiDB 集群一致的版本号（如 PD、TiKV），要么使用最新版本号（如 Alertmanager）。如果配置了该字段，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。请仅在需要使用某个固定组件的版本号时配置该参数。主要包含以下字段：
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:129]:
```suggestion
`component_versions` 用于指定某个组件的版本号。

- 如果没有配置 `component_versions`，各个组件要么使用和 TiDB 集群一致的版本号（如 PD、TiKV），要么使用最新版本号（如 Alertmanager）。
- 如果配置了该字段，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。

请仅在需要使用某个固定组件的版本号时配置该参数。

主要包含以下字段：
```

File: upgrade-tidb-using-tiup.md [hunk 7]
```diff
@@ -194,6 +194,27 @@ tiup cluster upgrade <cluster-name> v7.4.0
 >   3. reload 整个集群：`tiup cluster reload <cluster-name>`。此时，TiFlash 也会正常启动，无需额外操作。
 > - 在对使用 TiDB Binlog 的集群进行滚动升级过程中，请避免新创建聚簇索引表。
 
+#### 升级时指定组件版本
+
+从 tiup-cluster v1.14.0 开始，支持在升级集群的时候指定其中某些组件到特定版本。指定的这些组件会被固定到指定的版本，在下次升级中不会改变版本号，除非再次指定版本。
+
+{{< copyable "shell-regular" >}}
```
Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:201]:
```suggestion
```

File: tiup/tiup-cluster-topology-reference.md [hunk 8]
```diff
@@ -15,6 +15,7 @@ title: 通过 TiUP 部署 TiDB 集群的拓扑文件配置
 - [global](/tiup/tiup-cluster-topology-reference.md#global)：集群全局配置，其中一些是集群的默认值，可以在实例里面单独配置
 - [monitored](/tiup/tiup-cluster-topology-reference.md#monitored)：监控服务配置，即 blackbox exporter 和 node exporter，每台机器上都会部署一个 node exporter 和一个 blackbox exporter
 - [server_configs](/tiup/tiup-cluster-topology-reference.md#server_configs)：组件全局配置，可单独针对每个组件配置，若在实例中存在同名配置项，那么以实例中配置的为准
+- [component_versions](/tiup/tiup-cluster-topology-reference.md#component_versions)：组件版本号，当有组件使用和集群不一致的版本号时使用。tiup-cluster v1.14.0 引入该配置
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:18]:
```suggestion
- [component_versions](/tiup/tiup-cluster-topology-reference.md#component_versions)：组件版本号，当某个组件需要使用与集群不一致的版本时使用此配置。tiup-cluster v1.14.0 引入该配置
```

File: tiup/tiup-cluster-topology-reference.md [hunk 9]
```diff
@@ -36,6 +37,7 @@ title: 通过 TiUP 部署 TiDB 集群的拓扑文件配置
 - `group`：自动创建用户时指定用户所属的用户组，默认和 `<user>` 字段值相同，若指定的组不存在，则自动创建
 - `ssh_port`：指定连接目标机器进行操作的时候使用的 SSH 端口，默认值：22
 - `enable_tls`：是否对集群启用 TLS。启用之后，组件之间、客户端与组件之间都必须使用生成的 TLS 证书进行连接，默认值：false
+- `listen_host`：默认使用的 listen host。如果为空，每个实例会通过判断自身的 `Host` 字段是否包含 `:` 来自动设置成 `::` 或者 `0.0.0.0`。tiup-cluster v1.14.0 引入该配置
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:40]:
```suggestion
- `listen_host`：默认使用的监听 IP。如果为空，每个实例会根据其 `host` 字段是否包含 `:` 来自动设置为 `::` 或 `0.0.0.0`。tiup-cluster v1.14.0 引入该配置
```

File: tiup/tiup-cluster-topology-reference.md [hunk 10]
```diff
@@ -118,6 +120,43 @@ server_configs:
 
 上述配置指定了 TiDB 和 TiKV 的全局配置。
 
+### `component_versions`
+
+> **注意：**
+>
+> 对于 TiDB、TiKV、PD、TiCDC 等本身共用版本号的组件，没有完整的测试来保证它们能够在跨版本混部的场景下能够正常工作。请仅用于测试场景，或者在技术支持的帮助下进行操作。
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:127]:
```suggestion
> 对于 TiDB、TiKV、PD、TiCDC 等共用版本号的组件，尚未有完整的测试保证它们在跨版本混合部署的场景下能正常工作。请仅在测试场景或在[获取支持](/support.md)的情况下使用此配置。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 11]
```diff
@@ -118,6 +120,43 @@ server_configs:
...
+
+`component_versions` 用于指定某个组件的版本号。
+
+- 如果没有配置 `component_versions`，各个组件要么使用和 TiDB 集群一致的版本号（如 PD、TiKV），要么使用最新版本号（如 Alertmanager）。
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:131]:
```suggestion
- 如果没有配置 `component_versions`，各个组件默认使用与 TiDB 集群相同的版本号（如 PD、TiKV），或使用组件的最新版本号（如 Alertmanager）。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 12]
```diff
@@ -118,6 +120,43 @@ server_configs:
...
+- 如果配置了该字段，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:132]:
```suggestion
- 如果配置了该字段，相应的组件将固定使用指定的版本，并且在扩容或升级集群操作中继续使用该版本。
```

File: upgrade-tidb-using-tiup.md [hunk 13]
```diff
@@ -194,6 +194,25 @@ tiup cluster upgrade <cluster-name> v7.4.0
 >   3. reload 整个集群：`tiup cluster reload <cluster-name>`。此时，TiFlash 也会正常启动，无需额外操作。
 > - 在对使用 TiDB Binlog 的集群进行滚动升级过程中，请避免新创建聚簇索引表。
 
+#### 升级时指定组件版本
+
+从 tiup-cluster v1.14.0 开始，支持在升级集群的时候指定其中某些组件到特定版本。指定的这些组件会被固定到指定的版本，在下次升级中不会改变版本号，除非再次指定版本。
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:199]:
```suggestion
从 tiup-cluster v1.14.0 开始，支持在升级集群时使用 [`component_versions`](/tiup/tiup-cluster-topology-reference.md#component_versions) 指定某些组件的版本。该操作会将指定的组件在后续升级中保持固定版本，除非重新指定。

> **注意：**
>
> 对于 TiDB、TiKV、PD、TiCDC 等共用版本号的组件，尚未有完整的测试保证它们在跨版本混合部署的场景下能正常工作。请仅在测试场景或在[获取支持](/support.md)的情况下使用此配置。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 14]
```diff
@@ -118,6 +120,43 @@ server_configs:
...
+
+请仅在需要使用某个固定组件的版本号时配置该参数。
+
+主要包含以下字段：
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:136]:
```suggestion
`component_versions` 区块主要包含以下字段：
```

File: upgrade-tidb-using-tiup.md (hunk 13 above)
Comment (kaaaaaaang) [reply upgrade-tidb-using-tiup.md:199]: user cannot edit component_versions directly, user can only use flags to sepcify it

File: tiup/tiup-cluster-topology-reference.md (hunk 12 above)
Comment (kaaaaaaang) [reply tiup/tiup-cluster-topology-reference.md:132]: I think it is more clear to remain old description

File: upgrade-tidb-using-tiup.md (hunk 13 above)
Comment (Oreoxmt) [reply upgrade-tidb-using-tiup.md:199]: PTAL again
```
支持在升级集群时修改 TiDB 集群拓扑配置文件中 [`component_versions`](/tiup/tiup-cluster-topology-reference.md#component_versions) 配置指定某些组件的版本。
```

Comment (hfxsd) [reply upgrade-tidb-using-tiup.md:199]: 升级的时候是通过下面这些 -- 参数去指定的，不需要在此处链接到配置文件章节。配置文件仅用于部署和扩容使用。

Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:199]:
```suggestion
从 tiup-cluster v1.14.0 开始，支持在升级集群的时候指定其中某些组件到特定版本。指定的这些组件会被固定到指定的版本，在下次升级中不会改变版本号，除非再次指定版本。


> **注意：**
>
> 对于 TiDB、TiKV、PD、TiCDC 等共用版本号的组件，尚未有完整的测试保证它们在跨版本混合部署的场景下能正常工作。请仅在测试场景或在[获取支持](/support.md)的情况下使用此配置。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**